src/main/java/nl/rutgerkok/bo3tools/command/BO2ConvertFolderCommand.java

```diff
@@ -138,6 +131,39 @@ protected int convertBO2s(String author, CustomObjectCollection objects, String
         return count;
     }
 
+    /**
+     * Return a collection of all CustomObjects pertaining to the specified
+     * world, or global objects if no world is specified.
+     *
+     * Since OTG class CustomObjectCollection doesn't have methods to support
+     * iteration, this method accesses its collection fields by reflection.
+     *
+     * @param collection The OTG CustomObjectCollection containing the objects.
+     * @param worldName The name of the world whose objects are converted, or
+     *        null to convert global objects.
+     * @return the collection of CustomObjects.
+     */
+    @SuppressWarnings("unchecked")
+    protected List<CustomObject> getCustomObjects(CustomObjectCollection collection, String worldName) {
+        try {
+            if (worldName == null) {
+                Field field = collection.getClass().getDeclaredField("objectsGlobalObjects");
```

Comment:
It would be better to just ask @PG85 to make this collection iterable again. The fact that we have to use reflection to access Minecraft internals is OK: we cannot change Minecraft code. But for OpenTerrainGenerator reflection shouldn't be necessary.